openupgrade_scripts/apriori.py

```diff
@@ -35,12 +35,16 @@
     # OCA/account-financial-tools
     "stock_account_prepare_anglo_saxon_out_lines_hook": "stock_account",
     "account_menu": "account_usability",
+    # OCA/bank-statement-import
+    "account_statement_import_base": "account_statement_import",
     # OCA/hr
     "hr_recruitment_notification": "hr_recruitment",
     # OCA/hr-attendance
     "hr_attendance_user_list": "hr_attendance",
     # OCA/product-attribute
     "stock_account_product_cost_security": "product_cost_security",
+    # OCA/server-ux
+    "mass_operation_abstract": "mass_editing",
```

Comment:
not sure about this. please, could you give us some kind of proof?

Reply:
we have the dependency in v13: https://github.com/OCA/server-ux/blob/13.0/mass_editing/__manifest__.py#L15 but it's gone in v14: https://github.com/OCA/server-ux/blob/14.0/mass_editing/__manifest__.py#L15, while the module still exists.
It doesn't exist in v15 any more, that's why I added the merge there. Or we treat that as an abandoned module, then it will have to be cleaned up with database_cleanup

Reply:
I see.

![Selection_717](https://user-images.githubusercontent.com/25005517/211831506-39d89dd8-7e7c-4bdb-9743-36b86777553a.png)

My opinion then is not to merge and treat it as an abandoned module.

Reply:
agreed, removed that part